.github/workflows/shared-test.yml

```
@@ -175,6 +175,8 @@ jobs:
           parallel_mode: ${{ inputs.cpu_parallel_mode }}
           enable_coverage: ${{ inputs.enable_coverage }}
           gpu_tests: 'false'
+          # Internal pytest timeout 2 min below the step timeout so allure flushes before kill.
+          pytest_timeout_minutes: ${{ inputs.cpu_only_test_timeout_minutes - 2 }}
```

actionlint (1.7.12)

[error] got unexpected character ' ' while lexing integer part of number, expecting '0'..'9'

(expression)